test/navigation/OtherOfficialsNavigatorSpec.scala

```diff
@@ -379,13 +380,144 @@ class OtherOfficialsNavigatorSpec extends SpecBase {
       }
     }
 
-    "in Playback mode" when {
-      "attempting to go to any site" must {
-        "go to the SessionExpiredController page" in {
-          navigator.nextPage(AddAnotherOtherOfficialPage, PlaybackMode, emptyUserAnswers) mustBe
-            routes.SessionExpiredController.onPageLoad()
+    def goToPlaybackPage(index: Int): Call = index match {
+      case 0 => otherOfficialRoutes.AddedOneOtherOfficialController.onPageLoad()
+      case 1 => otherOfficialRoutes.AddedSecondOtherOfficialController.onPageLoad()
+      case 2 => otherOfficialRoutes.AddedThirdOtherOfficialController.onPageLoad()
+      case _ => routes.SessionExpiredController.onPageLoad()
+    }
+
+    List(0,1).foreach(index => {
```

Comment:
might have confused you when we had the call - please test that again index 2 (3rd other official) as well!